docs/blog/2018-11-01-hacktoberfest-wrapup/index.md

```diff
@@ -0,0 +1,91 @@
+
+We defined and started tackling our overall goals for Hacktoberfest:
+
+1. To provide a range of issues with a clearly defined scope, with minimal context needed outside of the issue description.
+2. To make contributing to Gatsby as welcoming, friendly, and accessible as we can -- especially to first-time contributors.
+
+We created a [master issue](https://github.com/gatsbyjs/gatsby/issues/8719) to be our Hacktoberfest "landing page", and thought of four general issue areas: OSS, Docs, Gatsby Store, and Ecosystem. Within that, our areas of greatest focus and attention were OSS and Docs.
+
+Most of the [OSS issues](https://github.com/gatsbyjs/gatsby/issues/8725) were existing issues that we identified as good candidates to surface to Hacktoberfest contributors.
+
+Most of the [Docs issues](https://github.com/gatsbyjs/gatsby/issues/7928) stemmed from an existing "content migration" effort, which aimed to repurpose valuable content already existing in the Gatsby blog, various plugin and starter READMEs, and other external resources, and pull it into the Gatsby docs.
+
+## Things that went well
+
+### 439 pull requests merged
+
+Thanks to everyone's contributions, throughout October we merged 439 pull requests from 250 different authors into the Gatsby repo!
+
+<figure>
+  <img alt="Github insights for the Gatsby repo, October 1 - November 1, 2018" height="400" src="./gatsby-hacktoberfest.png" />
+  <figcaption>
+    An awesome community response for Hacktoberfest!
+  </figcaption>
+</figure>
+
+### Over 170 new first-time contributors!
+
+Of the 250 different authors who contributed to this month, over 170 were first-time contributors to the Gatsby repo!
+
+<blockquote class="twitter-tweet" data-lang="en"><p lang="en" dir="ltr">150 new people started contributing to Gatsby last month! Welcome and thanks! <a href="https://twitter.com/hashtag/Hacktoberfest?src=hash&amp;ref_src=twsrc%5Etfw">#Hacktoberfest</a></p>&mdash; Gatsby (@gatsbyjs) <a href="https://twitter.com/gatsbyjs/status/1056235954824871936?ref_src=twsrc%5Etfw">October 27, 2018</a></blockquote>
+
+(We celebrated a bit early).
+
+### Closing out the content migration effort
+
+We were able to close out almost the entire [content migration effort](https://github.com/gatsbyjs/gatsby/issues/8103)! This was an absolutely amazing effort by the community, the value of which cannot be overstated.
+
+## Things to learn from
+
+### We were overwhelmed
+
+We’re a small team, and we weren’t fully prepared to handle the overwhelmingly enthusiastic response from the community. Many issues, particularly docs issues, were claimed within minutes of opening them. It slowed our response time, which was frustrating -- both for contributors, and for us.
+
+#### ➡️ We created codeowner teams
+
+To split our time more efficiently, we [created codeowner teams](with-great-contributions-comes-great-responsibility) to automatically notify Inkteam members when certain areas of review were required:
```

Comment:
Are people supposed to know what Inkteam means? Maybe just rephrase as team members or something?